cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
+            nargs='+',
+            help='The slug for the page you wish to delete'
+        )
+
+    def handle(self, *args, **options):
+        slug = options['slug'][0]
+        try:
+            page_to_delete = CFGOVPage.objects.get(slug=slug)
```

Comment:
This will raise an exception if more than one page exists with the given slug:

```py
>>> from v1.models import CFGOVPage
>>> CFGOVPage.objects.get(slug='agenda')
Traceback (most recent call last):
 ...
MultipleObjectsReturned: get() returned more than one CFGOVPage - it returned 2!
```